### PR TITLE
[ll] small Vulkan update

### PR DESCRIPTION
**Pull request by kvark**

### CODE REVIEW

File: src/backend/vulkan/src/command.rs [hunk 1]
```diff
@@ -206,7 +206,7 @@ impl command::RawCommandBuffer<Backend> for CommandBuffer {
                         region.src_subresource,
                         region.num_layers,
                     ),
-                    src_offset: offset,
+                    src_offset: offset.clone(),
```
Comment (msiglreith) [inline src/backend/vulkan/src/command.rs]: Ugh, `vkOffset3d` not implementing `Copy`.
We should address this in `ash` in the future.

File: src/backend/vulkan/src/lib.rs [hunk 2]
```diff
@@ -348,6 +348,11 @@ impl core::Adapter<Backend> for Adapter {
 
 #[doc(hidden)]
 pub struct RawDevice(pub ash::Device<V1_0>);
+impl fmt::Debug for RawDevice {
```
Comment (msiglreith) [inline src/backend/vulkan/src/lib.rs]: Does the compiler require it?

File: src/backend/vulkan/src/command.rs (hunk 1 above)
Comment (kvark) [inline src/backend/vulkan/src/command.rs]: I don't mind `Clone` either. `Copy` would work better if we need arrays of that stuff

File: src/backend/vulkan/src/lib.rs (hunk 2 above)
Comment (kvark) [inline src/backend/vulkan/src/lib.rs]: It does since I included `Arc<RawDevice>` in `DescriptorPool`

**Pull request merged**